.changes/next-release/bugfix-AWSSDKforJavav2-2a80679.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "anirudh9391",
+    "description": "Modify Enhanced Client to work on successfully updating nested attributes without loss of information"
```

Comment:
Can we add more information? Is there an associated GH issue? Suggesting something like below
> Introduce a new `TableSchema#itemMap` that fixes the issue where xxxx. See [xxx](github issue link). The existing xxx is now deprecated in favor of this new API.